libblkid/src/superblocks/bcache.c

```diff
@@ -12,8 +12,11 @@
 #include <stdio.h>
 
 #include "superblocks.h"
+#include "crc32c.h"
 #include "crc64.h"
+#include "xxhash.h"
 
+#define SECTOR_SIZE   512
```

Comment:
This sounds too common. What about BCACHEFS_SECTOR_SIZE? (like we use in other files, see hfs.c, iso9660.c)

Reply:
Done